python/ray/experimental/serve/global_state.py

```diff
@@ -43,8 +43,8 @@ def start_actor(self, actor_cls, tag, init_args=(), init_kwargs={}):
         self.actor_handles[handle] = tag
         return [handle]
 
-    def start_actor_with_creator(self, creator, tag):
-        handle = creator()
+    def start_actor_with_creator(self, creator, kwargs, tag):
```

Comment:
can we document what we expect the creator to do? before it was just a simple closure, now we actually need to be aware of its argument. 
```
Args:
    creator (Callable[Dict]): a closure that should return a newly crated actor handle when called with kwargs. The kwargs input is passed to `ActorCls_remote` method.
    ...
```